frontend/src/libs/components/header/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/enums.js';
+import { useAppDispatch, useCallback } from '~/libs/hooks/hooks.js';
+import { actions as authActions } from '~/slices/auth/auth.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  trigger: {
+    handleToggleModalOpen: () => void;
+    isOpen: boolean;
+  };
+};
+
+const DropDown: React.FC<Properties> = ({ trigger }) => {
+  const dispatch = useAppDispatch();
+
+  const handleSignOut = useCallback((): void => {
```

Comment:
I guess it would be better to name it as `handleLogout`, because you have a redux action and css class name using logout word, just to make it consistent